### PR TITLE
Follower's Surgical Ripper/Power Cells to fusion cells change.

### PR DESCRIPTION
**Pull request by Moribun**
This adds a custom sprite for a Surgical follower ripper. Sheeple was working on this and isabella and a few asked me if I could do it....which I did. it has 10 less damage than a standard ripper and is capable of being used for surgery. it has scalpel function when off. saw function when on and cauterizer when on. since its basically a super heated cosmic ripper. its coded to be craft able only by followers. I also added since Nep said it shouldnt be an issue and off duty follower donator loadout for ripper to spawn with one since he created this canonically and shared the Blueprint with the followers. aswell as some stuff boards wise to set up my own lab/clinic round start when i wanna do a chill round. 

EDIT: Also because im super dumb and I mean very very dumb my next work didnt go on my split branch sooooooooo SURPRISE fusion cell change too on this one..... glad I finished it tonight

## About The Pull Request
Edit: Changes sprites of power cells to fusion cells. and renames them
adds the recipe for the ripper
adds the requirement to the job role where only followers can make it
adds the ripper stats
adds the ripper sprites
adds donator loadout for character with it

## Why It's Good For The Game
edit: Because fusion cells was a thing in fallout. not bluespace shit
a few asked for this since in the older fallout games followers had rippers. it was also a proof of concept to see if I could get it to have no conflictions as a surgery tool. its all around weaker than a standard ripper but, it gives followers a unique melee option they lacked and even functions for surgery pretty effectively.

## Pre-Merge Checklist

- [ y ] You tested this on a local server.
- [ y ] This code did not runtime during testing.
- [ y ] You documented all of your changes.

## Changelog
Edit: changed sprites of power cells and names
Added surgical ripper code
Added sprites of ripper
Added donor loadout for creator

:cl:Moribund
add: Added new things: follower ripper sprites and coding
imageadd: added sprite for inhand and ground icon of surgical ripper

/:cl:

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
I'm going to say outright, while I love custom loadouts for the RP potential they provide, I need to emphasize something.
Since you've added this yourself, do be aware that if it's abused, I'm liable to outright remove it immediately.

Otherwise, this looks good.

**Comment by Moribun** [comment]
Yep its for off-duty clinic building nothing more when I wanna vibe. also currently working on ultracite rework for bluespace crystals replacement. im on alot of caffine and a roll currently

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 1]
```diff
@@ -266,3 +266,28 @@
...
+/obj/item/storage/box/large/custom_kit/ripper/PopulateContents()
+	new /obj/item/clothing/glasses/hud/health/eyepatch(src)// basic healthhud but cooler.
+	new /obj/item/melee/powered/surgripper(src)// easier than coding a skill-book giver for him to craft them
+	new /obj/item/clothing/suit/hooded/followerheavy(src)// crusader drip
+	new /obj/item/clothing/ears/headphones// hard to get for a fluff item
+	new /obj/item/clothing/head/f13/stalkerpelt(src)// morbid. but funny.
+	new /obj/item/storage/pill_bottle/chem_tin/fixer(src)// not craftable
+	new /obj/item/book/granter/trait/midsurgery(src)// skill level for surgery of basic doctor
+	new /obj/item/book/granter/trait/chemistry(src)// chem whiz
+	new /obj/item/stack/sheet/mineral/abductor// polymer for advanced surgery table
```
Comment (ProbablyCarl) [inline modular_citadel/code/modules/client/loadout/__donator.dm:284]: Going to request this removed, for now.

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 2]
```diff
@@ -266,3 +266,28 @@
...
+	new /obj/item/stack/sheet/mineral/silver// for surgery table ^
+	new /obj/item/disk/plantgene(src)// store plant data
+	new /obj/item/disk/plantgene(src)// store plant data
+	new /obj/item/circuitboard/machine/chem_master/advanced(src)//not a dispencer. splitter. can make patches so legion dont bitch.
+	new /obj/item/circuitboard/machine/gibber(src)// fetish reason dont ask.
+	new /obj/item/circuitboard/machine/plantgenes(src)// plant manipulator
+	new /obj/item/stock_parts/manipulator/femto(src)// for gibber, works with clothes on/faster.
+	new /obj/item/stock_parts/matter_bin/bluespace(src)// for gibber. gives most meat.
+	new /obj/item/circuitboard/machine/sleeper(src)// auto-doc
```
Comment (ProbablyCarl) [inline modular_citadel/code/modules/client/loadout/__donator.dm:293]: Going to request this removed, for now.

**Comment by RadiantFlash7** [comment]
I’m squinting a little at the loadout, mostly the plant manipulator. Wasteland doctor wasters don’t exactly get near that level of goodies. I’m not entirely sure why you’re intending on opening your own clinic as an offduty followers in the first place? The ripper is- also a little squint worthy, but if carl says it’s fine…

**Comment by ProbablyCarl** [comment]
Again, it's just a case of how it's utilized.
Easy enough to adjust if it harms a faction outright.

In any case, I'll merge this for now. Not test merging and just doing it outright, because it's liable that this'll stick.

**Pull request merged**